src/btree/bt_debug.c

```diff
@@ -1052,7 +1052,7 @@ __wt_debug_cursor_page(void *cursor_arg, const char *ofile)
     }
 
     WT_WITH_BTREE(
-      session, CUR2BT(cbt), ret = __wt_debug_page(session, NULL, cbt->ref, ofile, false));
+      session, CUR2BT(cbt), ret = __wt_debug_page(session, NULL, cbt->ref, ofile, true));
```

Comment:
Am curious as to why this is in this PR. @Siddhartha8899 was this related to the merge?

Reply:
try merging develop again, hopefully that helps.